roaring/container_stash.go

```diff
@@ -35,28 +36,85 @@ const (
 // less than 4,096 values, an array is often used. Containers with long runs of
 // integers would use run length encoding, and more random data usually uses
 // bitmap encoding.
+//
+// the names "nnn" and "typp" are temporary hacks to make failure to update
```

Comment:
yes, yes we should. they predate my realization that i specifically want those to never be accessed directly, because accessing them directly breaks nil containers.

Reply:
just a reminder that these should be renamed before merging

Reply:
okay, went through and did that. I removed every reference to them outside container_stash.go.

I sort of want to do an experiment where I split out the Container implementation into another package just to make sure I know what the interface to it is, but that wouldn't actually be a good design overall; the Container concept is pretty deeply tied to Roaring. I just want to make sure I'm not leaking things unintentionally.